docs/source/rule-testing.rst

```diff
@@ -73,43 +73,56 @@ For example, to replace a time based field with ``last_hour``::
     ]
   }
 
-Running Tests
-~~~~~~~~~~~~~~~~~~~~~~~
+  Running Tests
```

Comment:
We need a note explaining what warrants testing of what.

A normal user won't be sure if they add a new rule if they should only test the rule, the rule processor, or ...

I'd put something along the lines of "If you're only developing rules, you only need to run X. If you're developing StreamAlert, you should run Y" - assuming I understand this